scss/main.scss

```diff
@@ -93,14 +93,11 @@ a {
 }
 
 #main {
-  padding-bottom: 160px;
+  padding-top: 30px;
+  padding-bottom: 420px;
```

Comment:
Why is this necessary? Looks like `footer` has absolute positioning for some reason. Removing that negates the need for this, is that something that can be done?